1_slices_modules_and_objects/exercises/acronym/src/lib.rs

```diff
@@ -10,7 +10,15 @@
 /// ```
 // Side note: this doc example is also tested when running `cargo test`
 pub fn make_acronym(name: &str) -> String {
-    unimplemented!()
+    let mut acronym: String = String::new();
+    let split = name.split(' ');
+    
+    for i in split {
+        acronym.push(i.chars().nth(0).unwrap());
+    }
+
+    acronym.to_uppercase()
```

Comment:
This is perfectly valid, but can also be rewritten in a more functional manner:
```suggestion
    split.map(|word| word.chars().nth(0).unwrap().to_uppercase())
          .collect()
```